packages/components/src/products.ts

```diff
@@ -215,3 +212,12 @@ export const PRODUCTS: Record<
     primaryColor: '#000',
   },
 };
+
+export const sixHighlightedProducts = [
```

Comment:
nit: UPPER_SNAKE_CASE for constants
```suggestion
export const SIX_HIGHLIGHTED_PRODUCTS = [
```

Reply:
Good idea.